waspc/examples/todoApp/src/testing.ts

```diff
@@ -0,0 +1,84 @@
+import { prisma } from 'wasp/server'
+import type {
+  TestingAction,
+  VoidToNumberAuth,
+  VoidToNumberNoAuth,
+  BoolToNumberNoAuth,
+} from 'wasp/server/operations'
+import {
+  voidToNumberAuth as waspVoidToNumberAuth,
+  voidToNumberNoAuth as waspVoidToNumberNoAuth,
+  boolToNumberNoAuth as waspBoolToNumberNoAuth,
+  boolToNumberAuth as waspBoolToNumberAuth,
+} from 'wasp/server/operations'
+
+export const testingAction: TestingAction = async (args, context) => {
+  const result1 = await waspVoidToNumberNoAuth()
+  const result2 = await waspBoolToNumberNoAuth(true)
+
+  const user = context.user!
```

Comment:
If we are keeping these tests, I'd recommend using a conditional statement here so the action can work even without the user 🤷